app/views/admin/participants/_details.html.erb

```diff
@@ -0,0 +1,58 @@
+<table class="govuk-table">
+  <tbody class="govuk-table__body">
+    <tr class="govuk-table__row">
+      <th scope="row" class="govuk-table__header">Full name</th>
+      <td class="govuk-table__cell"><%= latest_induction_record.user.full_name %></td>
+      <td class="govuk-table__cell">
+        <%= govuk_link_to edit_name_admin_participant_path(latest_induction_record.participant_profile) do %>
+          Change <span class="govuk-visually-hidden">name</span>
+        <% end %>
+      </td>
+    </tr>
+    <tr class="govuk-table__row">
+      <th scope="row" class="govuk-table__header">Email address</th>
+      <td class="govuk-table__cell"><%= latest_induction_record.user.email %></td>
+      <td class="govuk-table__cell">
+        <%= govuk_link_to edit_email_admin_participant_path(latest_induction_record.participant_profile) do %>
+          Change <span class="govuk-visually-hidden">email</span>
+        <% end %>
+      </td>
+    </tr>
+    <tr class="govuk-table__row">
+      <th scope="row" class="govuk-table__header">ID</th>
+      <td class="govuk-table__cell"><%= latest_induction_record.user.id %></td>
+      <td class="govuk-table__cell"></td>
+    </tr>
+    <tr class="govuk-table__row">
+      <th scope="row" class="govuk-table__header">All email addresses associated with this user</th>
```

Comment:
```suggestion
      <th scope="row" class="govuk-table__header">Associated email addresses</th>
```

Or did @Lloyd-K suggest this content?

Reply:
No, haven't mentioned this to @Lloyd-K. Just a quick add in after the slack thread with Luke D. Happy to change to whatever's best